src/styles/blocks/button.scss

```
@@ -0,0 +1,31 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+
+  text-decoration: none;
+  font-weight: 700;
+  color: #fff;
```

don't forget to create variables for all colors

```suggestion
  color: $c-white;
```